approval_polls/views.py

```diff
@@ -30,12 +33,19 @@ def getPolls(request, poll_list, render_page):
       polls = paginator.page(paginator.num_pages)
   return render(request, render_page, {'latest_poll_list' : polls})
 
-class DetailView(generic.DetailView):
+class DetailView(generic.DeleteView):
   model = Poll
   template_name = 'approval_polls/detail.html'
   def get_queryset(self):
       return Poll.objects.filter(pub_date__lte=timezone.now())
 
+  @staticmethod
```

Comment:
Hi @Brianjr0428, this view does not only delete the poll but it also gets the poll. So it can not be a DeleteView. It will inherit from `generic.View` and have two methods `get` and `delete`